docs/webui_strings_explainer.md

```diff
@@ -0,0 +1,83 @@
+## WebUI Strings
+
+We have extended grit with the ability to automatically generate front end
+bindings for grit strings, and an associated Typescript enum.
+
+To expose a string to WebUI add a `formatter_data="webui=YourUi"` attribute
+in grit:
```

Comment:
nit: such examples typically named like `MyUi`
I would also name it `MyComponentUi` and give some examples that `MyComponent` is `AiChat` or `Wallet` or else

Reply:
I haven't given specific example, but I have linked the a PR in AIChat converting most of the strings over :smile: